dotcom-rendering/src/lib/thrift/nativeConnection.ts

```diff
@@ -115,7 +115,7 @@ export class NativeConnection<Context = void> extends ThriftConnection {
 		}
 	}
 
-	send(dataToSend: Buffer, context?: void | undefined): Promise<Buffer> {
```

Comment:
This signature is quite odd, but I can’t see why we wouldn’t be able to remove it…